README.md

```diff
@@ -26,3 +26,4 @@ The cursed mansion remained, a macabre sentinel in the forest, its secrets locke
 As she left the mansion not a word was heard no cars drove by not a person to be seen. When she arrived home her mum, dad and brother where not in which was strange it was late and her parents where never ones for late nights shurly they would have told here if they where going to be out. Never the less she went to bed for the night. The next day she woke up to the house still desseted she went on her phone the wifi was down and she had no signal what was happening.
 
 She decided she needed to investigate. Wondering through the dark gloomy forest Sarah searched and searched to find the mansion.  The wind whistled between the branches and shivers ran down her spine. She had escaped the mansion but the same darkness was still lingering in the atmosphere. Sarah's head darted back and forth as the scanned the area surrounding her. She needed to find somebody quick. Suddenly a light flashed between the barks of the trees, "Was someone else here?", Sarah wondered, maybe she wasn't alone. Maybe somebody could help her? Or perhaps somebody was after her...
+It was all a dream and she had woken up
```

Comment:
Is it just finishing without a full stop or exclamation point?

Reply:
Under 100 words :(